### PR TITLE
feat: Create header and mobile menu

### PR DESCRIPTION
**Pull request by eduAbreu**
Left as a draft for now for you to test and also I'm waiting for your PR to be merged to get the media queries.

### CODE REVIEW

File: components/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import MenuBurguer from '../../assets/icons/menu_burguer.svg';
```
Comment (kdabo) [inline components/Header/Header.js:4]: let's use tailwind's the most similar icons, so we don't import and create more assets? What do you think?

**Comment by kdabo** [comment]
let's also decide on the naming convention. I realised you are using uppercase and lower case for the folder names respectively. let's pick one. Which one do you prefer? 

Also, if we have only one file, is it really necessary to create folders?

File: components/Header/Header.js (hunk 1 above)
Comment (eduAbreu) [reply components/Header/Header.js]: Sure, I used the tailwind heroicons: https://github.com/tailwindlabs/heroicons

**Comment by eduAbreu** [comment]
@kdabo Added the desktop version. Please give it a look and let me know what you think of it.

**Pull request merged**